dist/tools/pic32prog/doc.md

```diff
@@ -0,0 +1,160 @@
+Flashing on Linux using pickit-3
+================================
+
+This document describes how to enable flashing a pic32-wifire from Linux with a
+PICkit3 using [pic32prog](https://github.com/sergev/pic32prog).
+
+It will require flashing a specific firmware on the PICkit3.
+As this can only be done from a Windows computer, that not many Linux users
+have, the following steps explain how to setup a Windows VirtualBox virtual
+machine and flash the PICkit3 from it.
+
+Informations come from this comment
+
+https://github.com/RIOT-OS/RIOT/pull/6092#issuecomment-261987955
+
+
+Steps
+-----
+
+### Setup your computer
+
+I ran this on Ubuntu 16.04
+
+* Install Virtualbox
+* Add yourself to the 'vboxusers' group
+  `sudo usermod -a -G vboxusers your_username`
+* Reboot
+
+### Setup Windows VM
+
+You can download a free 90 days valid windows virtualbox image from:
+
+    https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/
+
+I downloaded a Windows 10 virtualbox image
```

Comment:
Who is "I" (once people only see the markdown in the repo, people might wonder)?